openstudiocore/src/model/FanConstantVolume.cpp

```diff
@@ -148,6 +148,10 @@ namespace detail {
   }
 
   bool FanConstantVolume_Impl::setAvailabilitySchedule(Schedule& schedule) {
+    if (boost::optional<AirLoopHVAC> _airLoop = airLoopHVAC()) {
+      LOG(Info, briefDescription() << " is connected to an AirLoopHVAC '" << _airLoop->nameString()
+             << "', therefore its Availability Schedule will be overriden by the AirLoopHVAC's one.");
+    }
```

Comment:
I really like this reminder. I think people forget miss this fact. Too bad we can't hide the fan  avail schedule in the app IG, but I think for Zone equipment it still needs to be visible.

Reply:
I wanted to make it a Warn for that reason (I sometimes forget it too...) but thought it might clutter the openstudio-standards logs too much especially